frontend/src/screens/Map/TimeSeriesSlider.js

```diff
@@ -124,6 +124,10 @@ const TimeSeriesSlider = ({ availableDates, modifiedLayer }) => {
 
   const [currentValue, setCurrentValue] = useState(undefined);
 
+  useEffect(() => {
```

Comment:
When you have more layers in map visible with slider and deselect one of these layers, slider stay on the date you selected before on this layer. So I added this useEffect to always show LAST value. You can comment out this and check the behavior or if we should do it another way. In PublicMap.js we have code on LN142-144, if I comment this out, slider behave the same. What was the purpose of it? I suppose this should fetch the LAST values for the layer, but we have LAST values also without it, when you show the slider for the first time when we use it on LN214. (Thats because of LN132-134 in TimeSeriesSlider.js.) And when we use it on LN171 in PublicMap.js, this doesn't set slider to LAST, because currentValue already exist. So my question is should I remove PublicMap.js LN142-144? And leave this code in TimeSeriesSlider? Thanks for checking

Reply:
Hi, so this is ok?